services/airflow/src/lib/dag_helpers/dbt_helpers.py

```diff
@@ -23,6 +23,6 @@ def run_dbt_models(models: str, target_db_schema: str, **kwargs) -> None:
         ],
         capture_output=True,
         text=True,
+        check=True,
     )
     print(result.stdout)
-    print(result.stderr)
```

Comment:
since with `check=True` the subprocess raises an error with stderr and stdout the explicit print of stderr would never run (since empty when success and an error raised before when failure). stdout should still be printed in the event of success.